testing/upload_validation.ts

```diff
@@ -936,6 +936,19 @@ function validateFormulas(schema: z.ZodObject<any>) {
       },
       {message: 'A formula namespace must be provided whenever formulas are defined.', path: ['formulaNamespace']},
     )
+    .refine(
+      metadata =>
+        !(
+          metadata.defaultAuthentication?.type === AuthenticationType.CodaApiHeaderBearerToken &&
+          metadata.networkDomains?.some(
+            (domain: string) => !['coda.io', 'codahosted.io', 'localhost', 'calc-grpc-proxy'].includes(domain),
```

Comment:
`'localhost', 'calc-grpc-proxy'` are for the debug pack, which is the only pack that can use a non-public domain.